src/style.css

```diff
@@ -1 +1,31 @@
 /* add styles here */
+* {
+  margin: 0;
+}
+
+.body {
+  height: 160px;
+  width: 140px;
```

Comment:
Don't fix container size (if there is no such a requirement). Let the content size dictate it. To avoid overflow or accidental scroll bar. If there are no specific requirements for the body size, it's better to let its content define its size.